src/renderer/components/ft-video-player/ft-video-player.css

```diff
@@ -4,4 +4,42 @@
 
 .ftVideoPlayer {
   width:100%;
-}
\ No newline at end of file
+}
+
+.blocked {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  z-index: 2;
+  color: #ddd;
```

Comment:
Maybe the color and background-color should be decided based on theme?

Reply:
wouldnt that look weird though?

Reply:
Themed ones
<div>
<img src="https://user-images.githubusercontent.com/80553357/160259873-bf405c0c-7b53-4225-9f03-523b9ad98409.PNG" width=160>
<img src="https://user-images.githubusercontent.com/80553357/160259874-20551c91-6ab2-4ad5-bf82-2853a2bf28c3.PNG" width=160>
</div>

Reply:
Honestly can't tell which is better

Reply:
I will go with black and white (video-js background is always #000) Is that alight?

Reply:
Yeah sure go for it!